site/class-1-play/bjorn2123.svg

```diff
@@ -0,0 +1,52 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+  <!--Update the title with your username-->
+  <title>@Bjorn2123</title>
+
+    <circle
+            cx = "120"
+            cy = "120"
+            r = "100"
+            />
+
+    <rect
+          x = "250"
+          y = "20"
+          height="200"
+          width="400"
+          stroke="red"
+          stroke-width="20"
+    />
+    
+    <line 
+          x1 = "140"
+          y1="300"
+          x2="180"
+          y2="500"
+          stroke="aqua"
+          stroke-width="13"
+          />
+    
+   
+    
+   <g
+      stroke="antiquewhite"
+      stroke-width="20"
+      >
+       <polygon points="250,60 300,400 90,300"/>
+  <path d="M150 0 L75 200 L225 200 Z" />
+ </g> 
+    
+     <text x="350" y="150" fill="blue" >I love SVG!</text>
+<svg width="500" height="200" viewBox="0 0 50 40" >
```

Comment:
You’re still opening another `<svg>` element here, and closing two `<svg>` elements at the end of the file.
Could you remove one? It’s like the `<html>` element for SVG: you can only have one.

Then, make sure to use the `viewBox` attribute on the first `<svg>`.

Also: add a `font-size` attribute on any element!